ibis-server/app/model/data_source.py

```
@@ -218,15 +218,25 @@ def get_oracle_connection(info: OracleConnectionInfo) -> BaseBackend:
+                private_key=info.private_key.get_secret_value(),
+                **info.kwargs if info.kwargs else dict(),
+            )
+        else:
+            return ibis.snowflake.connect(
+                user=info.user.get_secret_value(),
+                password=info.password.get_secret_value(),
+                account=info.account.get_secret_value(),
+                database=info.database.get_secret_value(),
+                schema=info.sf_schema.get_secret_value(),
+                **info.kwargs if info.kwargs else dict(),
+            )
```

Pylint (3.3.7)

[refactor] Unnecessary "else" after "return", remove the "else" and de-indent the code inside it

(R1705)